src/game/AI/ScriptDevAI/scripts/world/npcs_special.cpp

```diff
@@ -2805,6 +2805,206 @@ bool GossipHello_npc_gossip_npc(Player* player, Creature* creature)
     return true;
 }
 
+struct PTRPaymasterAI : public ScriptedAI
```

Comment:
This seems useless.

Reply:
Might be. I'm not sure if I can have the gossip functions hooked without an AI. If I can then I can throw this out